forms-web-app/src/controllers/appellant-submission/appeal-statement.js

```diff
@@ -25,32 +26,54 @@ exports.postAppealStatement = async (req, res) => {
     return;
   }
 
+  if (body['does-not-include-sensitive-information'] !== 'i-confirm') {
+    res.redirect(`/${VIEW.APPELLANT_SUBMISSION.APPEAL_STATEMENT}`);
+    return;
+  }
+
   const { appeal } = req.session;
-  const task = appeal[sectionName][taskName];
+  let ap;
+
+  try {
+    appeal.sectionStates[sectionName][taskName] = getTaskStatus(appeal, sectionName, taskName);
+    ap = await createOrUpdateAppeal(appeal);
```

Comment:
in every other controller, this would be:

`req.session.appeal = await createOrUpdateAppeal(appeal);`